extra-utils/genie/autobuild/build

```diff
@@ -0,0 +1,4 @@
+cd "$SRCDIR"/genie
+
+make build
```

Comment:
Add some `abinfo` here please.

Reply:
Done

Reply:
Please separate `abinfo` for different build steps.

Reply:
Done